includes/class-mailchimp-woocommerce-service.php

```diff
@@ -640,20 +640,29 @@ public function setLandingSiteCookie()
         if ($this->getLandingSiteCookie() != false) return $this;
 
         $http_referer = $this->getReferer();
-
-        if (!empty($http_referer)) {
+        error_log("server". $_SERVER['REQUEST_URI'] );
+        if (!empty($http_referer) && !wp_doing_ajax() ) {
 
             // grab the current landing url since it's a referral.
             $landing_site = home_url() . wp_unslash($_SERVER['REQUEST_URI']);
 
             $compare_refer = str_replace(array('http://', 'https://'), '', $http_referer);
             $compare_local = str_replace(array('http://', 'https://'), '', $landing_site);
 
+            error_log("referer". $http_referer);
```

Comment:
@alejandrogiv let's remove all these error logs - only or only if the debugging is set to true.